autopush/tests/test_web_validation.py

```diff
@@ -324,10 +324,12 @@ def test_no_headers(self):
         schema.context["settings"].router.get_uaid.return_value = dict(
             router_type="webpush",
         )
-        data = self._make_test_data(body="asdfasdf",
-                                    headers={"ttl": "invalid"})
-        result, errors = schema.load(data)
-        eq_(errors, {'headers': {'ttl': [u'Not a valid integer.']}})
+        data = self._make_test_data(body="asdfasdf")
+
+        with assert_raises(InvalidRequest) as cm:
+            schema.load(data)
+
+        eq_(cm.exception.message, "Unknown Content-Encoding")
```

Comment:
might as well check for the expected errno and maybe status_code here additionally